docs/elements.md

```diff
@@ -499,7 +499,7 @@ Fill in the blank field that allows for mathematical symbol input.
 
 **question.html**
 ```html
-<pl-symbolic-input answers-name="symbolic_math" variables="x, y" label="$z =$"></pl-symbolic-input>
+<pl-symbolic-input answers-name="symbolic_math" variables="x, y" extra-functions="loglog, zeta" label="$z =$"></pl-symbolic-input>
```

Comment:
I don't know if `loglog` is a good example here, since it's equivalent to `log log` (if your definition is the same as mine) and could lead to confusion. Maybe something generic like `f`?

Reply:
@jonatanschroeder 
I am putting ``loglog`` in as an alternate in a new logarithms function category. I use ``\log\log x`` *A LOT* in my research these days. I feel it justifies a special non-default definition. I will work on making a better set of test cases though.

Reply:
The reason I'm suggesting not to use loglog is because, if a user decides to use `log log` instead, this would typically be correct (if I'm not mistaken), but would not be considered equivalent since `loglog` would be considered a custom function.

Reply:
@jonatanschroeder 
This is a small point. I would like to see that function because I love it so much and like the syntax of having `loglog` instead of `log log`. I can just implement a work around to give myself the option of that stylistic and syntactic ANT sugar if I need it. I really am not up for a long debate about which functions are worthy of inclusion in the new groups of function types getting put into the PR.